src/provider/Provider.js

```diff
@@ -0,0 +1,14 @@
+export class Provider {
```

Comment:
I like that you're improving the modularity of their code; I hope they appreciate it